### PR TITLE
zh-translation: docs/root/configuration/http/http_filters/buffer_filter.rst

### PR DESCRIPTION
**Pull request by zucong**
ref: https://github.com/cloudnativeto/envoy/issues/265

- [x] Docs

### CODE REVIEW

File: docs/root/configuration/http/http_filters/buffer_filter.rst [hunk 1]
```diff
@@ -1,22 +1,18 @@
 .. _config_http_filters_buffer:
 
-Buffer
+缓冲区
 ======
 
-The buffer filter is used to stop filter iteration and wait for a fully buffered complete request.
-This is useful in different situations including protecting some applications from having to deal
-with partial requests and high network latency.
+缓冲区过滤器被用于停止过滤器迭代以及等待完整缓冲的请求。这在很多情况下很有用，包括以免某些应用处理不完整的分块请求和高网络延迟的情况。
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/buffer_filter.rst]:
```suggestion
缓冲区过滤器被用于停止过滤器迭代以及等待完整缓冲的请求。这在很多情况下很有用，包括避免某些应用处理不完整的分块请求和高网络延迟。
```

File: docs/root/configuration/http/http_filters/buffer_filter.rst [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
 
-If enabled the buffer filter populates content-length header if it is not present in the request
-already. The behavior can be disabled using the runtime feature
-`envoy.reloadable_features.buffer_filter_populate_content_length`.
+如果启用缓冲区过滤器，在请求头中没有 content-length 头，则缓冲区过滤器会添加这个头。
+这一行为可以使用运行时属性 `envoy.reloadable_features.buffer_filter_populate_content_length` 来禁用。
 
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.buffer.v3.Buffer>`
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/buffer_filter.rst]:
```suggestion
* :ref:`v3 API 参考 <envoy_v3_api_msg_extensions.filters.http.buffer.v3.Buffer>`
```

**Review by addozhang — COMMENTED** [review]
很棒的翻译，添加了点建议。

**Review by addozhang — APPROVED** [review]
/LGTM

File: docs/root/configuration/http/http_filters/buffer_filter.rst [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 
-The buffer filter is used to stop filter iteration and wait for a fully buffered complete request.
-This is useful in different situations including protecting some applications from having to deal
-with partial requests and high network latency.
+缓冲区过滤器被用于停止过滤器迭代以及等待完整缓冲的请求。这在很多情况下很有用，包括避免某些应用处理不完整的分块请求和高网络延迟。
 
-If enabled the buffer filter populates content-length header if it is not present in the request
-already. The behavior can be disabled using the runtime feature
-`envoy.reloadable_features.buffer_filter_populate_content_length`.
+如果启用缓冲区过滤器，在请求头中没有 content-length 头，则缓冲区过滤器会添加这个头。
```
Comment (majinghe) [inline docs/root/configuration/http/http_filters/buffer_filter.rst]:
```suggestion
如果启用缓冲区过滤器，在请求中没有 content-length 头，则缓冲区过滤器会添加这个头。
```
这样更通顺点。

**Comment by majinghe** [comment]
@zucong 就一点建议，感觉加上更通顺。

File: docs/root/configuration/http/http_filters/buffer_filter.rst (hunk 3 above)
Comment (zucong) [reply docs/root/configuration/http/http_filters/buffer_filter.rst]: 确实

**Pull request merged**